packages/components/fields/creatable-select-field/src/creatable-select-field.tsx

```diff
@@ -0,0 +1,453 @@
+   */
+  'aria-label'?: ReactSelectCreatableProps['aria-label'];
+  /**
+   * HTML ID of an element that should be used as the label (for assistive tech)
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  'aria-labelledby'?: ReactSelectCreatableProps['aria-label'];
+  /**
+   * 	Focus the control when it is mounted
+   */
+  isAutofocussed?: boolean;
+  /**
+   * Remove the currently focused option when the user presses backspace
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  backspaceRemovesValue?: ReactSelectCreatableProps['backspaceRemovesValue'];
+  /**
+   * Map of components to overwrite the default ones, see [what components you can override](https://react-select.com/components)
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  components?: ReactSelectCreatableProps['components'];
+  /**
+   * Custom method to filter whether an option should be displayed in the menu
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  filterOption?: ReactSelectCreatableProps['filterOption'];
+  /**
+   * The id to set on the SelectContainer component
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  containerId?: ReactSelectCreatableProps['id'];
+  /**
+   * Is the select value clearable
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isClearable?: ReactSelectCreatableProps['isClearable'];
+  /**
+   * Is the select disabled
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isDisabled?: ReactSelectCreatableProps['isDisabled'];
+  /**
+   * Is the select read-only
+   */
+  isReadOnly?: ConstrainBooleanParameters;
+  /**
+   * Override the built-in logic to detect whether an option is disabled
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isOptionDisabled?: ReactSelectCreatableProps['isOptionDisabled'];
+  /**
+   * Support multiple selected options
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isMulti?: ReactSelectCreatableProps['isMulti'];
+  /**
+   * Whether to enable search functionality
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isSearchable?: ReactSelectCreatableProps['isSearchable'];
+  /**
+   * Indicates the input field has an error
+   */
+  hasWarning?: boolean;
+  /**
+   * Maximum height of the menu before scrolling
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  maxMenuHeight?: ReactSelectCreatableProps['maxMenuHeight'];
+  /**
+   * 	Dom element to portal the select menu to
```

Comment:
```suggestion
   * Dom element to portal the select menu to
```

Remove extra spaces.